coaster/utils.py

```diff
@@ -662,6 +662,7 @@ def deobfuscate_email(text):
     Deobfuscate email addresses in provided text
     """
     text = unescape(text)
+    text = sanitize_html(text, strip=False)
```

Comment:
I'm not sure `sanitize_html` is the right method to call here. What we're looking for is a test for `<a href="mailto:test@example">this</a>`, which the new regex will catch. Sanitization not needed (it also slows down the validator).